cluster-sharding/src/main/scala/akka/kubernetes/sample/ClusterStateRoute.scala

```diff
@@ -22,13 +22,16 @@ object ClusterStateRoute extends ClusterHttpManagementJsonProtocol {
     path("cluster" / "members") {
       get {
         complete {
-          val readView = ClusterReadViewAccess.internalReacView(cluster)
+          val readView = ClusterReadViewAccess.internalReadView(cluster)
           val members = readView.state.members.map(memberToClusterMember)
 
-          val unreachable = readView.reachability.observersGroupedByUnreachable.toSeq.sortBy(_._1).map {
-            case (subject, observers) ⇒
-              ClusterUnreachableMember(s"${subject.address}", observers.toSeq.sorted.map(m ⇒ s"${m.address}"))
-          }
+          val unreachable = readView.reachability.observersGroupedByUnreachable.toSeq
```

Comment:
ClusterReadViewAccess is needed because the full `reachability` is internal. Can we use some other public API in the example to not promote usage of internal APIs to end users?

Reply:
Let's do https://github.com/akka/akka-management/issues/434 for 1.0 then we can remove this